cmd/auth/auth.go

```diff
@@ -20,6 +20,7 @@ import (
 
 var (
 	backend     string
```

Comment:
Thanks for the contribution! Can't we re-use the existing backend variable?
```
removeCmd.Flags().StringVarP(&b, "backends", "b", "", "Backend AI providers to remove (separated by a comma)")
```

This looks good - might just need to make sure it can work for both remove/add?

Reply:
> Can't we re-use the existing backend variable?

Sure thing, I thought it might be a little confusing to use the same variable, but I'm happy to go with that.

> might just need to make sure it can work for both remove/add?

I will add a comment with the log from the verification I did locally.

Reply:
@AlexsJones I updated the PR. Please see the log from the local verification in the [PR description](https://github.com/k8sgpt-ai/k8sgpt/pull/711#issue-1947088422)